world.js

```diff
@@ -110,12 +111,24 @@ World.prototype.locate = function(point) {
   return ind
 }
 
+World.prototype.cues = function() {
+  var cues = []
+  this.tiles.forEach(function (tile) {
+    var cue = _.find(tile.children, function(child) {return child.props.cue})
```

Comment:
Does this only search tiles children? What about searching the children of tiles children too in a recursive fashion - or no need?